docs/migrate-to-managed-asm/virtual-service.yaml

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 # [START servicemesh_migrate_to_managed_asm_virtual_service_virtualservice_frontend_ingress]
-# This VirtualService routes the example.com traffic from my-gateway
-# to port 80 of the hello-world-service.
+# This VirtualService splits HTTP traffic coming into the cluster you're migrating away from.
+# This VirtualService ensures that the traffic will be shared between both clusters.
```

Comment:
I made this error because I copied the [VirtualService from a different repo](https://github.com/NimJay/intro-to-istio-ingress-gateway/blob/main/virtual-service.yaml).
I forgot to update the comment.